### ISSUE TITLE
[Content] HLR: Introduction Page Copy Updates

### ISSUE THREAD

**Issue by luke-gcio** [department-of-veterans-affairs/va.gov-team#5622]
## Purpose
Copy updates based on usability findings and AMO feedback.

### Background
Full research plan and results are [here](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/decision-reviews/higher-level-review/research).

* **Is the request related to a VA.gov tool? Provide a brief description of the tool or functionality:**  
   * This is part of the appeals process.  Higher Level Review on VA.gov pulls the appeal out of the legacy process and moves it onto the new process.
* **Provide links to any available collateral: product briefs/outlines, wireframes, prototypes, user research synthesis:** 
   * Specific copy editing task for Higher Level Review introduction page based on the findings from [HLR Research](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/decision-reviews/higher-level-review/research/research-summary.md)
    * Research readout [deck](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/decision-reviews/higher-level-review/research/HLR%20usability%20testing%20-%20Research%20Readout.pdf)
    * Product outline for [appeals](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/decision-reviews)

### Request Type
- [x] **Content support for authenticated app/tool.**
   * Word document with draft if available, or link to the URL where the draft content appears:
![image](https://user-images.githubusercontent.com/53824125/73979879-4d06e280-48ec-11ea-9215-b0cb7bba4d5c.png)
   * Finding summary
![image.png](https://images.zenhubusercontent.com/5d54025346ffca000148e7f7/8ff3b667-a8f2-4606-b01e-b7178efedc7b)
   * InVision Page: [File an HLR](https://vsateams.invisionapp.com/share/W5U21BDFTQK#/screens/404076957_Signed_In_-_File_An_HLR)
   * Provide design context: 
      * This is the landing page that begins the HLR flow.
      * The research showed that we should explain Opt In and Opt Out significantly better - right now both AMO and veteran research wanted us to define what it means to Opt into this new process.
      * Recommendation: Consider adding an info box about the Opt out process, or adding a F.A.Q. section (why should I do this, what are the benefits, what are the implications, what the difference between new and old appeals process? etc.) 
      * This will require some discussions with the designers.
   * Scope: How many screens?
      * One screen for this ticket.  We will break each page down into individual tickets for easier tracking.

### Desired/target implementation date
* End of Sprint 16.  We will not be launching until some time after that sprint.

### Tasks
- [ ] Meet with researcher for full context
- [ ] Provide updated copy
- [ ] Profit

### Requestor info
- Team name: Benefits and Memorials 1
- DEPO product owner: Andrea Schneider
- Product manager: Luke Majewski
- Researcher: Christian Valla
- Designer: Kevin Stachura
- FE engineer: Robin Garrison
- Product/team Slack channel: #vsa-benefits-memorials

**Comment by MickinSahni** [comment]
2020-02-07 PW Standup
- Hi @lunascoop we're still working through the intake process but do you/we have all the info we need in this ticket before we can commit in future sprint(s)?

**Comment by luke-gcio** [comment]
Thanks @MickinSahni.  One note, this will for sure require a meeting with our researcher in order to obtain all the info.  I captured what I could in the description but 30 minutes with the researcher will provide way more context and should be a "task" in this story.

**Comment by MickinSahni** [comment]
Thanks, @luke-gcio. Feel free to add that task. Once @lunascoop has a chance for an initial triage pass, we'll likely look to you/your researcher to set that convo up.

**Comment by jenniferlee-dsva** [comment]
@MickinSahni CC @luke-gcio @lwalsh617 @lunascoop  - I believe this is a DUPLICATE ticket for work that was previously submitted to us here: 
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/4139

Ticket #4139 is currently blocked/on hold until the HLR tool itself is ready to launch. Please connect with @AZSchneider  - and let us know when the tool is ready.
